packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -180,9 +180,12 @@ export default class SalesforceCRMService implements CRM {
   };
 
   private salesforceCreateEvent = async (event: CalendarEvent, contacts: Contact[]) => {
+    const log = logger.getSubLogger({ prefix: [`[salesforceCreateEvent]:${event.uid}`] });
+
     const appOptions = this.getAppOptions();
 
     const writeToEventRecord = await this.generateWriteToEventBody(event);
+    log.info(`Writing to event fields: ${Object.keys(writeToEventRecord)} `);
```

Comment:
We're seeing `System.NullPointerException: Attempt to de-reference a null object` which may mean there's a field that's getting past the check here. Need to confirm with logs